frontend/packages/core/src/Table/metadata-table.tsx

```diff
@@ -76,12 +76,19 @@ const TableCell = styled(MuiTableCell)({
 });
 
 const Grid = styled(MuiGrid)({
+  display: "flex",
   ".MuiFormControl-root": {
+    flexDirection: "row",
+  },
+  ".MuiFormControl-root .MuiInputBase-root": {
     height: "40px",
     width: "100px",
-    flexDirection: "row",
+    alignSelf: "center",
+  },
+  ".MuiFormControl-root .MuiFormHelperText-root.Mui-error": {
+    flex: 1,
```

Comment:
to solve for this issue when there are longer errors
<img width="298" alt="Screen Shot 2021-01-06 at 5 00 55 PM" src="https://user-images.githubusercontent.com/39421794/103823801-c64d0f00-5040-11eb-8ebb-969e77a57965.png">